backend/tests/Designer.Tests/Controllers/OrgTextController/GetLanguagesTests.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Designer.Tests.Controllers.ApiTests;
+using Designer.Tests.Utils;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Xunit;
+
+namespace Designer.Tests.Controllers.OrgTextController;
+
+public class GetLanguagesTests : DesignerEndpointsTestsBase<GetLanguagesTests>, IClassFixture<WebApplicationFactory<Program>>
+{
+    public GetLanguagesTests(WebApplicationFactory<Program> factory) : base(factory)
+    {
+    }
+
+    private const string DeveloperName = "testUser";
+    private const string OrgName = "ttd";
+
+    [Fact]
+    public async Task GetLanguages_ReturnsOk_WithAllLanguages()
```

Comment:
Missing http status code in test name
```suggestion
    public async Task GetLanguages_Returns200OK_WithAllLanguages()
```